src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/project.json

```diff
@@ -1,8 +1,4 @@
 {
-  "supports": {
```

Comment:
We probably need to add reference to a package that defines these, in order to enable this validation. I think we'll be able to do that if we move to DNX meta-packages (once we update to IC v37). We can then also avoid manually listing FX dependencies.